src/react-slideshow-image.d.ts

```diff
@@ -0,0 +1 @@
+declare module 'react-slideshow-image';
```

Comment:
<img width="713" alt="image" src="https://user-images.githubusercontent.com/26671751/185794966-9b456dd9-849f-4a47-afb7-d163fba3f836.png">

what a shame. Good idea to just create a declaration file.